### PR TITLE
Add director to the Directors report

### PR DESCRIPTION
**Pull request by faisal-amin82**
- Add directors to the director report
- Add unit tests to the directors report controller for the add director functionality
- Add section to the add or remove directors page for the add director

Resolves: SFA-2373

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>uk.gov.companieshouse</groupId>
     <artifactId>company-accounts.web.ch.gov.uk</artifactId>
-    <version>unversioned</version>
+    <version>749c8df9</version>
```
Comment (BSaunders95) [inline pom.xml]: Please can you revert this?

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java [hunk 2]
```diff
@@ -52,7 +63,7 @@ public String getAddOrRemoveDirectors(@PathVariable String companyNumber,
 
         addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
 
-        AddOrRemoveDirectors addOrRemoveDirectors = new AddOrRemoveDirectors();
+      //  AddOrRemoveDirectors addOrRemoveDirectors = new AddOrRemoveDirectors();
```
Comment (BSaunders95) [inline src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java]: Should this be removed?

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java [hunk 3]
```diff
@@ -17,25 +21,33 @@
...
 public class AddOrRemoveDirectorsController extends BaseController implements ConditionalController {
 
+
     @Autowired
     private HttpServletRequest request;
 
     @Autowired
     private DirectorService directorService;
 
+    AddOrRemoveDirectors addOrRemoveDirectors = new AddOrRemoveDirectors();
```
Comment (divya051988) [inline src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java]: Should this be private AddOrRemoveDirectors addOrRemoveDirectors with @Autowired annotation ?

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java [hunk 4]
```diff
@@ -17,25 +21,33 @@
...
+
     private static final UriTemplate URI =
             new UriTemplate("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/add-or-remove-directors");
 
     private static final String ADD_OR_REMOVE_DIRECTORS = "addOrRemoveDirectors";
+    private static final String DIRECTOR_TO_ADD = "directorToAdd";
```
Comment (divya051988) [inline src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsController.java]: Add a new line above please .

**Pull request closed without merging**